documentation/slate/source/includes/authentication.md

```diff
@@ -1,7 +1,7 @@
 Authentication
 ==============
 
-> 3 ways to request Navitia
+> 4 ways to request Navitia
```

Comment:
This is going to be a good test for @benoit-bst's github action script. To validate that the new publication script works well :)